deps/exokit-bindings/webglcontext/src/webgl.cc

```diff
@@ -626,6 +626,12 @@ void setGlConstants(T &proto) {
   JS_GL_SET_CONSTANT("PIXEL_PACK_BUFFER_BINDING" , 0x88ED);
   JS_GL_SET_CONSTANT("PIXEL_UNPACK_BUFFER_BINDING", 0x88EF);
 
+  // OVR_multiview2 in draft
+  JS_GL_SET_CONSTANT("FRAMEBUFFER_ATTACHMENT_TEXTURE_NUM_VIEWS_OVR", 0x9630);
```

Comment:
Does this need to be global? I'm not sure if the spec says it is, or if it's only on the extension.

Reply:
Probably not, it should be through OVR_multiview2, will fix.

Reply:
8b2c577
and
8cc91fd